proto/opamp.proto

```diff
@@ -621,19 +621,31 @@ enum AgentCapabilities {
     // Add new capabilities here, continuing with the least significant unused bit.
 }
 
-// The health of the Agent.
-message AgentHealth {
-    // Set to true if the Agent is up and healthy.
+// The health of the Agent and sub-components
```

Comment:
Should we mark this with "// Status: [Beta]" since it was just changed? That way when we declare 1.0 we still have a chance to fix this message in case we made any mistakes here.